app/src/main/java/org/schulcloud/mobile/data/local/PreferencesHelper.java

```diff
@@ -70,6 +70,15 @@ public String saveCurrentUsername(String username) {
         return null;
     }
 
+    public String saveCurrentSchoolId(String schoolId) {
+        SharedPreferences.Editor editor = mPref.edit();
+        editor.putString("currentSchool", schoolId);
+
+        if (editor.commit())
+            return schoolId;
+        return null;
```

Comment:
I don't like the error handling here. Why do you return the same string? Simply `return editor.commit()` as boolean if you don't want to throw an exception. Also, in which cases can the `commit()` method fail anyway?